### DISCUSSION TITLE
O que e por onde começar?

### DISCUSSION THREAD

**Discussion by JevPac** [github/brasil#106] [General]
essa pergunta é tanto por mim que quer começar em WEB e GAMEDEV, então so vou soltar essa pergunta aqui em busca de alguma ajuda, ou melhor uma direção para qual alguem possa me apontar

**Comment by grsantos56** [comment]
Olá! Tentar abraçar Web e Gamedev ao mesmo tempo logo no início pode te sobrecarregar. Aqui vai uma rota prática:

**1. Use o JavaScript como ponte**
Comece pelo básico da Web (HTML, CSS e **JavaScript**). O JS é o terreno comum ideal: serve tanto para criar sites quanto para desenvolver jogos de navegador usando frameworks como **Phaser.js** (2D) ou **Kaboom.js**.

**2. Quer ir para Gamedev "Puro" depois?**
Pule para a engine **Godot** (leve e excelente para iniciantes) ou **Unity** (usa a linguagem C#, que também é absurdamente forte para mercado de Backend na Web).

**Dica de ouro:** Escolha apenas uma área pelos próximos 3 meses. Faça uma página web simples ou um clone do jogo "Pong". Dar um passo de cada vez é o segredo. Boa sorte na jornada!
